build/llamafile-convert

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+FILE=$1
+TYPE=${2:-both}
+SCRIPTNAME=${0##*/}
+
+if [ -z "$FILE" ]; then
+  echo "Usage: $SCRIPTNAME <gguf file or url> [cli|server|both]"
```

Comment:
Could you remove the `[cli|server|both]` argument? The `llamafile` and `llamafile-server` commands are now unified at head. It's no longer needed.